packages/groq/package.json

```diff
@@ -26,17 +26,20 @@
   "license": "MIT",
   "author": "Sanity.io <hello@sanity.io>",
   "sideEffects": false,
+  "type": "module",
   "exports": {
     ".": {
-      "source": "./src/groq.ts",
-      "import": "./lib/groq.mjs",
-      "require": "./lib/groq.js",
+      "require": "./lib/groq.cjs",
       "default": "./lib/groq.js"
     },
+    "./define": {
+      "require": "./lib/define.cjs",
+      "default": "./lib/define.js"
+    },
```

Comment:
```suggestion
```
I would prefer we don't add this export at first, if it turns out we can't have our cake and eat it too we can always add it later 🙌

Reply:
Oops that's me forgetting to remove that export. I've fixed up the PR.